TileStache/Mapnik.py

```diff
@@ -109,7 +124,10 @@ def __init__(self, layer, mapfile, fonts=None, scale_factor=None):
     def prepareKeywordArgs(config_dict):
         """ Convert configured parameters to keyword args for __init__().
         """
-        kwargs = {'mapfile': config_dict['mapfile']}
+        try:
```

Comment:
This can be accomplished more easily like so:
`config_dict.get('mapconfig') or config_dict.get('mapfile')`